pysiaf/aperture.py

```diff
@@ -645,17 +648,34 @@ def plot_detector_origin(self, frame, which='both'):
             Which detector origin to plot: 'both', 'det', 'sci'
```

Comment:
possibly add `raw` as option. (Then we would need to convert `both` to `all`) or supply a list.